packages/core/stories/tutorials/combo-charts.ts

```diff
@@ -0,0 +1,114 @@
+The charts that can be seamlessly combined are all different **axis charts only**. This is the **reccommended way to use \`ComboCharts\`**.
+
+Supported \`ChartTypes\` to combine:
+- \`LineChart\`
+- \`ScatterChart\`
+- \`StackedScatterChart\`
+- \`AreaChart\`
+- \`StackedAreaChart\`
+- \`BarChart\`
+- \`GroupedBarChart\`
+- \`StackedBarChart\`
+
+Below is an example of combining a carbon-charts \`AreaChart\` with a \`LineChart\`.
+
+\`\`\`
+const verySimpleComboData = [
+    { group: "Dataset 1", value: 650, date: "Tuesday" },
+	{ group: "Dataset 2", value: 296, date: "Tuesday" }
+	...more data
+];
+
+const comboOptions = {
+	...otherOptions // global options apply to all the charts listed in chartTypes (where relevant)
+	axes: {...}
+	comboChartTypes: [ // takes an array of objects
+		{
+			type: "area" // use a valid ChartType from enums
+			options: {}, // component specific options (i.e, turn on the scatter points, adjust scatter radius)
+			correspondingDatasets: ["Dataset 1"] // the datasets that we want to map with this type of chart
+		},
+		{
+			type: "line" // use a valid ChartType from enums
+			options: {
+				points: {
+					enabled: false; // disable the point on the LineChart won't affect the points in the AreaChart
+				}
+			}, // component specific options
+			correspondingDatasets: ["Dataset 2"]
+		}]
+};
+\`\`\`
+
+See the storybook demos for more combinations of charts and configurations.
+
+____
+
+### Using custom graphs & components
+Custom components and **independent** carbon-charts components can also be rendered into a \`ComboChart\`. This allows
+users to create their own components and combine them with the pre-existing componentry offered.
+
+Below is an example of combining a carbon-charts \`AreaChart\` with a standalone \`Line\` class constructor.
+
+**Reminder: \`ComboChart\` extends axis chart functionality and should only comprise of graphs/components that use axes.**
+
+
+\`\`\`
+// import standanlone componentry
+import {
+	Line
+} from ".././../src/components/index";
+import { options } from ".././../src/configuration"; // need include base options for the carbon-chart if you are using a component separately
+
+const verySimpleComboData = [
+    { group: "Dataset 1", value: 650, date: "Tuesday" },
+	{ group: "Dataset 2", value: 296, date: "Tuesday" }
+	...more data
+];
+
+const comboOptions = {
+	...otherOptions // global options apply to all the charts listed in chartTypes (where relevant)
+	axes: {...}
+	comboChartTypes: [ // takes an array of objects
+		{
+			type: "area" // use a valid ChartType from enums
+			options: {},
+			correspondingDatasets: ["Dataset 1"] // the datasets that we want to map with this type of chart
+		},
+		{
+			type: Line // using an imported class constructor
+			// add the base options for that type of chart if it is in the carbon-charts library and any additional configurations
+			options: Tools.merge({}, options.lineChart,  {
```

Comment:
there's no `Tools` imported here. either import from the library or use `Object.assign()`